iceoryx_posh/include/iceoryx_posh/iceoryx_posh_types.hpp

```diff
@@ -131,7 +131,7 @@ constexpr uint32_t MAX_PROCESS_NUMBER = 300U;
 /// This limitation is coming due to the fixed capacity of the cxx::vector (This doesn't limit the offered number of
 /// instances)
 /// @todo #415 increase number back to 50 once service registry is available via shared memory
-constexpr uint32_t MAX_NUMBER_OF_SERVICES = 10U;
+constexpr uint32_t MAX_NUMBER_OF_SERVICES = 100U;
```

Comment:
I think you can remove the `@todo` above?

Reply:
Quite the contrary, more todos await in #1074